overlord/snapstate/snapstate.go

```diff
@@ -67,7 +67,15 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 			return nil, err
 		}
 		if model == nil || model.Base() == "" {
-			return nil, fmt.Errorf("cannot install snapd snap on a model without a base snap yet")
+			var experimentalAllowSnapd bool
+			tr := config.NewTransaction(st)
+			err := tr.Get("core", "experimental.snapd-snap", &experimentalAllowSnapd)
```

Comment:
Should this no be "system" (I don' know where the translation happens). The code is fine as-is but for "nice" factor.